subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/codecs/Codecs.kt

```diff
@@ -294,6 +294,8 @@ class Codecs(
 
         bind(EnumCodec)
         bind(RegexpPatternCodec)
+        bind(UrlCodec)
+        bind(UriCodec)
```

Comment:
❓ Why is `UriCodec` necessary?

Reply:
I added it for symmetry but missed the fact that it is serializable. Now removed it.
Thanks for the good catch